cmd/server-main.go

```diff
@@ -144,18 +144,20 @@ func initConfig() {
 
 // Generic Minio initialization to create/load config, prepare loggers, etc..
 func minioInit(ctx *cli.Context) {
-	// Is TLS configured?.
-	globalIsSSL = isSSL()
-
-	// Initialize minio server config.
-	initConfig()
-
 	// Enable all loggers by now so we can use errorIf() and fatalIf()
 	enableLoggers()
```

Comment:
Yes. But some those inter functions are used at run time also.  We can't log them into console later.  I can fix wherever possible.